src/backend/src/services/change-requests.services.ts

```diff
@@ -621,4 +621,51 @@ export default class ChangeRequestsService {
       data: { dateDeleted: new Date(), deletedBy: { connect: { userId: submitter.userId } } }
     });
   }
+
+  /**
+   * set reviewers to the given change request
+   * @param submitter The user who sets a reviewer to the change request
+   * @param userId The reviewer who reviews the change request
+   * @param crId The change request that will be reviewed
+   */
+  static async requestCRAReview(submitter: User, userIds: number[], crId: number) {
+    const reviewers = await getUsers(userIds);
+
+    // check if any reviewers' role is below leadership
+    const underLeads = reviewers.filter((user) => !isLeadership(user.role));
+    const underLeadsIds = underLeads.map((reviewer) => reviewer.userId);
+
+    if (underLeads.length > 0)
+      throw new AccessDeniedException(
+        `User(s) with the following ids are not at least in a leadership: ${underLeadsIds.join(', ')}`
```

Comment:
change to users with the following name. user ids mean nothing to the user

Reply:
also we really shouldn't expose raw ids to the user